pyCGM_Single/pyCGM.py

```diff
@@ -2321,6 +2776,91 @@ def JointAngleCalc(frame,vsk):
                                                     [[L_Xaxis_x,L_Xaxis_y,L_Xaxis_z],
                                                     [L_Yaxis_x,L_Yaxis_y,L_Yaxis_z],
                                                     [L_Zaxis_x,L_Zaxis_y,L_Zaxis_z]]]]
+
+    Parameters
+    ----------
+    frame : dict 
+        Dictionaries of marker lists.
+            { [], [], [], ... }      
+    vsk : dict, optional
+        A dictionary containing subject measurements from a VSK file.
+            { [], [], [], ... }
+    
+    Returns
+    -------
+    r, jc : tuple 
+        Returns r, the result of all the joint calculations, 
+        and jc, a dictionary for joint centers.
+    
+    Example
+    -------
+    >>> import numpy as np
+    >>> from math import *
+    >>> from .pyCGM import *
+
+    >>> frame = {'RHEE': np.array([406.46331787, 227.56491089,  48.75952911]),
```

Comment:
I appreciate the effort here, but is there a way to use the helper functions of loading the data instead of typing it all out?

Reply:
I used helper functions instead of typing it all out for this function. Considering how this was the longest doctest that I did, do you want me to reformat all similar doctests using the helper functions or leave them as they are?

Reply:
If I recall, the other doctests are maximum 4 lines? I think that is okay, just ones that are many lines long we would want to use the helper.